consensus/istanbul/backend/message_senders.go

```diff
@@ -103,24 +104,26 @@ func (sb *Backend) Gossip(payload []byte, ethMsgCode uint64) error {
 		}
 	}
 
-	sb.asyncMulticast(peersToSendMsg, payload, ethMsgCode)
-
-	return nil
+	return sb.asyncMulticast(peersToSendMsg, payload, ethMsgCode)
 }
 
 // sendMsg will asynchronously send the the Celo messages to all the peers in the destPeers param.
-func (sb *Backend) asyncMulticast(destPeers map[enode.ID]consensus.Peer, payload []byte, ethMsgCode uint64) {
+func (sb *Backend) asyncMulticast(destPeers map[enode.ID]consensus.Peer, payload []byte, ethMsgCode uint64) error {
 	logger := sb.logger.New("func", "AsyncMulticastCeloMsg", "msgCode", ethMsgCode)
-
+	payload2, err := rlp.EncodeToBytes(payload)
```

Comment:
nit: `payloadEncoded` or `rlpPayload` seems more meaningful.

Reply:
I'll change it